studio/model.py

```diff
@@ -43,12 +43,14 @@ def __init__(self, key, filename, args, pythonenv,
                  time_last_checkpoint=None,
                  time_finished=None,
                  info={},
+                 env={},
```

Comment:
I am not sure I get the point of adding env to the experiment - the contents of the config will be put into queue message json as well, with env variables expanded. That way we 1) do not store our AWS keys with experiments in the database and 2) store only variables that are needed.